pkg/registry/keeps_test.go

```diff
@@ -67,6 +70,67 @@ func TestRegisterSigner(t *testing.T) {
 			persistenceMock.persistedGroups[0],
 		)
 	}
+
+	signer2, err := newTestSigner(1)
+	if err != nil {
+		t.Fatalf("failed to get signer: [%v]", err)
+	}
+
+	err = kr.RegisterSigner(keepAddress1, signer2)
+
+	expectedError := fmt.Errorf("signer for keep [%s] already registered", keepAddress1.String())
+	if err == nil || (!reflect.DeepEqual(expectedError, err)) {
+		t.Errorf(
+			"unexpected error\nexpected: [%v]\nactual:   [%v]",
+			expectedError,
+			err,
+		)
+	}
```

Comment:
Can we make it a separate test case, e.g. `TestRegisterSignerDuplicate`? We don't need to do any assertions for the first signer there, just confirm we get this error.